cfme/configure/access_control/__init__.py

```diff
@@ -1118,7 +1121,7 @@ def copy(self, name=None):
         view.toolbar.configuration.item_select('Copy this Role to a new Role')
         view = self.create_view(AddRoleView, wait=10)  # implicit assert
         new_role = self.parent.instantiate(name=name)
-        view.fill({'name_txt': new_role.name})
+        view.fill({'name_txt': new_role.name, "vm_restriction_select": restriction})
```

Comment:
**QUESTION:** Is it valid to pass `None` for restriction? if not, we might want to add a check.